webapp/src/Serializer/ContestProblemVisitor.php

```diff
@@ -46,5 +59,28 @@ public function onPostSerialize(ObjectEvent $event): void
             );
             $visitor->visitProperty($property, $color);
         }
+
+        // Problem statement
+        if ($contestProblem->getProblem()->getProblemtextType() === 'pdf') {
+            $route = $this->dj->apiRelativeUrl(
+                'v4_app_api_problem_statement',
+                [
+                    'cid' => $contestProblem->getContest()->getApiId($this->eventLogService),
+                    'id'  => $contestProblem->getApiId($this->eventLogService),
+                ]
+            );
+            $property = new StaticPropertyMetadata(
+                ContestProblem::class,
+                'statement',
+                null
+            );
+            $visitor->visitProperty($property, [
+                [
+                    'href'     => $route,
+                    'mime'     => 'application/pdf',
+                    'filename' => 'statement.pdf',
```

Comment:
Can we use the letter here, say A.pdf, B.pdf and so on?

Reply:
Done. Note that the filename is mainly used for people who want to create a Contest Package from an API and the spec says files should be placed in `<endpoint>/<id>/<filename>` so in this case `problems/A/A.pdf`. But I think thats fine, even better.